src/vorta/scheduler.py

```diff
@@ -333,24 +333,26 @@ def reload_all_timers(self):
             self.set_timer_for_profile(profile.id)
 
     def next_job(self):
-        next_job = now = dt.now()
-        next_profile = None
-        for profile_id, timer in self.timers.items():
-            if timer['type'] != ScheduleStatusType.SCHEDULED:
-                continue
-
-            if next_job == now and timer['dt'] > next_job and timer['qtt'].isActive():
-                next_job = timer['dt']
-                next_profile = profile_id
-            elif next_job > now and timer['dt'] < next_job and timer['qtt'].isActive():
-                next_job = timer['dt']
-                next_profile = profile_id
-
-        if next_profile is not None:
-            profile = BackupProfileModel.get_or_none(id=next_profile)
-            return f"{next_job.strftime('%H:%M')} ({profile.name})"
-        else:
-            return self.tr('None scheduled')
+        now = dt.now()
+
+        def is_scheduled(timer):
+            return (
+                timer["type"] == ScheduleStatusType.SCHEDULED
+                and timer["qtt"].isActive()
+                and timer["dt"] >= now
+            )
+
+        next_job = min([(profile_id, timer['dt']) for profile_id, timer in self.timers.items() if is_scheduled(timer)], key=lambda i: i[0], default=None)
```

Comment:
Now the line is too long :grimacing:  

```suggestion
        timers = [
                    (profile_id, timer["dt"])
                    for profile_id, timer in self.timers.items()
                    if is_scheduled(timer)
                ]
                next_job = min(timers, key=lambda i: i[0], default=None)
```